Lab 2

vladimirsah opened the pull request.

vladimirsah committed:
Create Program

vladimirsah committed:
Update Program

vladimirsah committed:
Update Program

vladimirsah committed:
Update Program

NemirovichMark commented:
Hello Vladimir.

Parse(Console.ReadLine()); - kills your program. Protect it!

2.1 Height should be positive (just logic)

3.11 Marks should be in the range from 2 to 5.
![image](https://user-images.githubusercontent.com/112826355/195831876-48565813-0601-46d2-b559-9ea737f7fe2c.png)

3.12 & 3.13 Use switch-operator as task requires.

vladimirsah committed:
Update Program

Исправил недочеты

NemirovichMark commented:
Hello Vladimir.

1.1 1.7 2.1 2.4 3.4 3.11 3.12 3.13
InvalidInputValue:
 Parse(Console.ReadLine());

3.11 Marks should be in the range from 2 to 5.

3.12 3.13
Solve it with switch-operator as the task requires.

3.13 
if (x == 0) { S = a * b; } - should be positive
if (x == 2) { S = a / 4 * Math.Sqrt(4 * b * b - a * a); } - InvalidInputValue (when sqrt(-10))

vladimirsah committed:
Update Program

Исправил всё

NemirovichMark commented:
Hello Vladimir.

task1_1, task1_7, task2_4, task3_4, task3_12
 Использование локальной переменной "x", которой не присвоено значение

task3_11 Использование локальной переменной "x1", которой не присвоено значение
Also you sum marks like -1231 

task3_13
 Использование локальной переменной "n", которой не присвоено значение

vladimirsah committed:
Update Program

Я убрал случайно попавшее в код условие, из-за которого программа не работала. Однако я не понял вашего замечания, в программу нельзя ввести 1 или число, большее 5

NemirovichMark commented:
Hello Vladimir.

![image](https://user-images.githubusercontent.com/112826355/197780166-5a5fdb4a-be60-4d4f-aeb0-cc4b0511a20c.png)

And it often occur, when no normal students (out by break)

![image](https://user-images.githubusercontent.com/112826355/197779895-92fe1858-bbd0-4c9c-8e12-b7fbdccd40b8.png)

vladimirsah committed:
Update Program

NemirovichMark commented:
Nice. 1 line)

You can come to class to defend 2nd lab.
You can make a fork of next lab and send the request to your branch:
https://github.com/NemirovichMark/3rd_Lab
